build/xtask/src/clippy.rs

```diff
@@ -41,8 +41,7 @@ pub fn run(
                 println!();
             }
             println!(
-                "================== {} [{}] ==================",
-                name, crate_name
+                "================== {name} [{crate_name}] ==================",
```

Comment:
is this now no longer 80 characters long?

Reply:
78 characters, just slipping under the line